web/src/app/users/UserContactMethodList.js

```diff
@@ -118,20 +107,21 @@ export default function UserContactMethodList(props) {
     if (!cm.disabled) {
       actions.push({
         label: 'Send Test',
-        onClick: () =>
+        onClick: () => {
           sendTest({
             variables: {
               id: cm.id,
             },
-          }),
+          })
+          setShowSendTestByID(cm.id)
```

Comment:
If we always show the dialog, maybe we can just send the message from there?

We could see when the last message was sent and even indicate to the user the 1 minute cooldown (e.g. with a retry button or something)